src/services/apps-metadata.ts

```diff
@@ -0,0 +1,62 @@
+/**
+ * Copyright (c) 2024, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+import { PredefinedProperties } from '../utils/types';
+
+// https://github.com/gridsuite/deployment/blob/main/docker-compose/docker-compose.base.yml
+// https://github.com/gridsuite/deployment/blob/main/k8s/resources/common/config/apps-metadata.json
+export type Metadata = MetadataCommon | MetadataStudy;
+export type Url = string | URL;
+
+export function fetchEnv() {
+    return fetch('env.json').then((res) => res.json());
+}
```

Comment:
```suggestion
export type EnvJson = {
    appsMetadataServerUrl?: Url;
    mapBoxToken?: string;

    // https://github.com/gridsuite/deployment/blob/main/docker-compose/env.json
    // https://github.com/gridsuite/deployment/blob/main/k8s/live/azure-dev/env.json
    // https://github.com/gridsuite/deployment/blob/main/k8s/live/azure-integ/env.json
    // https://github.com/gridsuite/deployment/blob/main/k8s/live/local/env.json
    [key: string]: string;
};

export async function fetchEnv(): Promise<EnvJson> {
    return (await fetch('env.json')).json();
}
```

Reply:
If `[key: string]: string;` is uncommented it won't work

Reply:
didn't know that typescript asked that all values are compatible type...